src/Turbo/Mercure/Broadcaster.php

```diff
@@ -25,14 +25,15 @@
  *
  * Supported options are:
  *
- * * topics (string[]) Mercure topics to use, defaults to an array containing the Fully Qualified Class Name with "\" characters replaced by ":" characters.
- * * createTemplate (string) The Twig template to render when a new object is created
- * * updateTemplate (string) The Twig template to render when a new object is updated
- * * removeTemplate (string) The Twig template to render when a new object is removed
- * * private (bool) Marks Mercure updates as private
- * * id (string) ID field of the SSE
- * * type (string) type field of the SSE
- * * retry (int) retry field of the SSE
+ *  * topics (string[]) Mercure topics to use, defaults to an array containing the Fully Qualified Class Name with "\" characters replaced by ":" characters.
+ *  * createTemplate (string) The Twig template to render when a new object is created
+ *  * updateTemplate (string) The Twig template to render when a new object is updated
+ *  * removeTemplate (string) The Twig template to render when a new object is removed
+ *  * private (bool) Marks Mercure updates as private
+ *  * id (string) ID field of the SSE
+ *  * type (string) type field of the SSE
+ *  * retry (int) retry field of the SSE
+ *  * transports (string[])
```

Comment:
this one is added to the list, this is a stub for now

Reply:
As `template`, it could be a generic argument added to the attribute's constructor.